config/src/test/java/io/atomix/core/config/jackson/JacksonConfigProviderTest.java

```diff
@@ -47,4 +48,24 @@ public void testYaml() throws Exception {
     assertEquals("test", config.getClusterConfig().getName());
     assertEquals(1, config.getPrimitives().get("foo").getSerializerConfig().getTypes().size());
   }
+
+  @Test
+  @Ignore
```

Comment:
why ignore? add env.yaml to src/test/resources?

Reply:
Can't set an environment variable in a junit test